src/app/shared/services/skin/skin.service.ts

```diff
@@ -1,48 +1,57 @@
 import { Injectable } from "@angular/core";
 import { BehaviorSubject } from "rxjs";
 import { LocalStorageService } from "src/app/shared/services/local-storage/local-storage.service";
-import { APP_CONSTANTS } from "src/app/data";
+import { APP_CONFIG } from "src/app/data";
+import { APP_SKINS } from "src/skins";
+import { IAppSkin } from "src/skins/skin.model";
 
-const { APP_FIELDS } = APP_CONSTANTS;
-
-// TODO - Where should these live and be added to? Should ultimately come from sheets?
-// Could live at deployment level, e.g. app_constants.compatible_skins
-const SKINS = ["default", "modular"];
+// TODO - The skin service needs these values, but getting them from the
+// appDataService would cause a circular dependency. Possible solution: limit
```

Comment:
Ideally the appConfigService shouldn't import anything from the skin service (see previous comment), which should hopefully resolve the circular dependency issue